core/src/test/resources/nelson/canopus_cronJob.mustache

```diff
@@ -0,0 +1,56 @@
+apiVersion: batch/v1beta1
```

Comment:
Naming of this file - can we be consistent between snake case and camel case? (snake case prefered here)

Reply:
Ah yes good call :-) It was originally `canopus_cronjob` and I renamed it on a whim completely forgetting it used to at least have a naming convention